daml-triggers/daml/Trigger/AutoApprove.daml

```diff
@@ -134,3 +138,11 @@ getStrictMarker m1@AutoApproveTransferProposalMarker { autoApproveType = Limited
 getStrictMarker m@AutoApproveTransferProposalMarker { autoApproveType = LimitedMaxAmount maxAmount } _ = m
 getStrictMarker _ m@AutoApproveTransferProposalMarker { autoApproveType = LimitedMaxAmount maxAmount } = m
 getStrictMarker m _                                                                                    = m
+
+
+instance Semigroup Blacklist where
```

Comment:
You could have implemented Monoid and use mconcat, couldn't you?

Reply:
No, not really. That was the idea, but then I realised I cannot implement `mempty`, because I do not know the party upfront.

Reply:
meh